src/Components/Blazor/DevServer/src/Commands/ServeCommand.cs

```diff
@@ -20,11 +23,28 @@ public ServeCommand(CommandLineApplication parent)
 
             HelpOption("-?|-h|--help");
 
+            ApplicationPath = new CommandArgument()
+            {
+                Description = "Path to the client application dll",
+                MultipleValues = false,
+                Name = "<PATH>",
+                ShowInHelpText = true
+            };
+            Arguments.Add(ApplicationPath);
+
             OnExecute(Execute);
         }
 
+        public CommandArgument ApplicationPath { get; private set; }
+
         private int Execute()
         {
+            if (string.IsNullOrWhiteSpace(ApplicationPath.Value))
+            {
+                throw new InvalidOperationException($"Invalid path {ApplicationPath.Value}.");
```

Comment:
This error will be pretty unclear if it's actually an empty string. It will just say `Invalid path `.

```suggestion
                throw new InvalidOperationException($"Invalid value for parameter '{nameof(ApplicationPath)}'. Value supplied: '{ApplicationPath.Value}'");
```